### PR TITLE
Fixed Emergency Shuttle now always playing its hyperspace warm-up sound

### PR DESCRIPTION
**Pull request by DeityLink**
The way timeleft is calculated causes the countdown to generally skip every other second, thus it skips the second at which the sound is supposed to start playing.

:cl:
* bugfix: The Emergency Shuttle should now always be playing its hyperspace warm up sound.

### CODE REVIEW

File: code/controllers/shuttle_controller.dm [hunk 1]
```diff
@@ -357,7 +357,7 @@ datum/emergency_shuttle/proc/process()
 
 		if(1)
 
-			if(timeleft == 5)
+			if(timeleft == 6 || timeleft == 5)
```
Comment (clusterfack) [inline code/controllers/shuttle_controller.dm]: Why not check if timeleft <= 6 and then put another var that prevents it from firing the sound again

Comment (DeityLink) [reply code/controllers/shuttle_controller.dm]: I considered doing that, but I tested this solution and it worked fine, so I told myself the less code lines/vars the better.

Comment (clusterfack) [reply code/controllers/shuttle_controller.dm]: Doing less lines of code is what gave you this problem in the first place t b h

Comment (DeityLink) [reply code/controllers/shuttle_controller.dm]: touché

**Pull request merged**